### ISSUE TITLE
程序越运行越慢的问题

### ISSUE THREAD

**Issue by MichaelYONGHENG** [cqu-wb/MTTV#1]
你好，我想请教一下关于程序训练时越运行越慢的问题

我先正常完成了特征提取，然后使用这个命令开始训练
python train.py --task fakeddit --label_type 2_way_label --batch_sz 256 --gradient_accumulation_steps 1 --max_epochs 20 --name fakeddit_2_way --bert_model bert-base-uncased --global_image_embeds 5 --region_image_embeds 20 --num_image_embeds 25 --n_workers 8

我发现在程序刚开始的时候显卡的利用率很高，程序运行的很快，但是越往后显卡的利用率越低，但显存是占着的（只是不计算），程序运行的很慢，想请教一下这个问题正常吗？

**Comment by cqu-wb** [comment]
你好，我们在运行过程没有出现过类似的问题。您提供的信息也无法定位问题，建议提供更详细的错误信息。

**Comment by MichaelYONGHENG** [comment]
> 你好，我们在运行过程没有出现过类似的问题。您提供的信息也无法定位问题，建议提供更详细的错误信息。

你好，没有报错信息，程序可以正常运行，但是第一个epoch的前四分钟很快，GPU的利用率也很高，但是之后就变得很慢，一个epoch需要3个小时，请问这样正常吗

**Comment by MichaelYONGHENG** [comment]
因为我设置的gradient_accumulation_steps=1，所以我把训练代码改成了这样：

with tqdm(total=len(train_loader), postfix={'Loss': 0.0}) as pbar:
            for batch in train_loader:
                txt, segment, mask, img, tgt, regions = batch
                txt, img = txt.cuda(), img.cuda()
                mask, segment = mask.cuda(), segment.cuda()
                regions = regions.cuda()
                out = model(txt, mask, segment, img, regions)
                tgt = tgt.cuda()
                loss = criterion(out, tgt)
                pbar.set_postfix({"Loss" : loss.item()})
                pbar.update(1)
                train_losses.append(loss.item())
                loss.backward()
                optimizer.step()
                optimizer.zero_grad()
观察到在四分钟后out = model(txt, mask, segment, img, regions)的时间显著增加，我并没有在网络里看到有关于时间t的处理，所以感觉很疑惑

**Comment by JinruYang** [comment]
你好，我的bert一直报这个错误Model name 'bert-base-uncased' was not found in model name list找不到原因可以帮忙解答一下吗

**Comment by JinruYang** [comment]
> 因为我设置的gradient_accumulation_steps=1，所以我把训练代码改成了这样：
> 
> with tqdm(total=len(train_loader), postfix={'Loss': 0.0}) as pbar: for batch in train_loader: txt, segment, mask, img, tgt, regions = batch txt, img = txt.cuda(), img.cuda() mask, segment = mask.cuda(), segment.cuda() regions = regions.cuda() out = model(txt, mask, segment, img, regions) tgt = tgt.cuda() loss = criterion(out, tgt) pbar.set_postfix({"Loss" : loss.item()}) pbar.update(1) train_losses.append(loss.item()) loss.backward() optimizer.step() optimizer.zero_grad() 观察到在四分钟后out = model(txt, mask, segment, img, regions)的时间显著增加，我并没有在网络里看到有关于时间t的处理，所以感觉很疑惑

请问您在复现代码的时候发现weibo数据集中有大部分新闻是没有图片的这一问题吗，请问您是如何处理的，能否提供一个完整图像的数据集
